neurom/analysis/morphtree.py

```diff
@@ -264,6 +264,40 @@ def trunk_length(tree):
         return 0.0
 
 
+def trunk_vector(tree, soma):
+    '''Vector of trunk defined as (initial tree point - soma center)
+       of the tree.
+    '''
+    return mm.vector(tree.value, soma.center)
+
+
+def trunk_elevation(tree, soma):
```

Comment:
I though you were referring to the trunk_elevation and trunk_azimuth (see your comment above) and those should be the same. If your question refers to trunk_direction or initial_direction I am open to more changes depending on what everyone prefers. After all it is a matter of taste (and definition) as you put it and I don't have a problem with either of the proposed names. I would prefer to keep trunk_elevation and trunk_azimuth though.